io_scene_dae/export_dae.py

```diff
@@ -232,12 +232,15 @@ def export_material(self, material, double_sided_hint=True):
             fxid, material.name))
         self.writel(S_FX, 2, "<profile_COMMON>")
 
-        # Find and fetch the textures and create sources
+        # Find and fetch the textures and create sources    
         sampler_table = {}
         diffuse_tex = None
         specular_tex = None
         emission_tex = None
         normal_tex = None
+        
+        #TODO, use Blender 2.8 principled shader and connected maps
```

Comment:
Was this overlooked? It doesn't export references to textures or what am i missing?